Dockerfile-pachd-skaffold

```diff
@@ -0,0 +1,25 @@
+# syntax = docker/dockerfile:experimental
+# Build the manager binary
+FROM golang:1.12.5 as builder
+
+WORKDIR /workspace
+
+COPY . .
+
+RUN --mount=type=cache,target=/go/pkg/mod \
```

Comment:
This isn't a comment so much as a question—I've never seen `RUN` with no commands before. Does it use the `ENTRYPOINT` of the `golang:1.12.5` image or something? Is this effectively running `go build ... src/server/cmd/pachd/main.go`?

Finally, my slight preference would be to run `go build ... src/server/cmd/pachd` (i.e. build the whole package) if possible, just because I see no reason not to do it, but `go build ... src/.../pachd/main.go` only works if there are no other go files in the package. But then I'm not even sure that that's the command being run here...

Reply:
This is actually all one long command (up to line 17), happy to change to build the package instead